saas/fsm/redundant_apps/fsm-redundant_apps.pt

```diff
@@ -0,0 +1,124 @@
+name "SaaS Manager - Redundant Apps"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will create an incident when Flexera SaaS Manager identifies application categories with an excessive number of applications. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/redundant_apps/) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
+severity "low"
+category "SaaS Management"
+default_frequency "weekly"
+info(
+  version: "1.0",
```

Comment:
should be version 2.0 when the policy uses the Cred Mgr.

Reply:
Applied updated version and updated root README